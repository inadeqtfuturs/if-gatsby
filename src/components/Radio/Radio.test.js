import React from 'react';
import { Radio } from '@components';
import ThemeRender from '@utils/testing';

const mockItems = [
  { name: 'home', value: '/' },
  { name: 'nest', value: '/' }
];

describe('Menu', () => {
  it('renders correctly', () => {
    const tree = ThemeRender(
      <Radio label="test" name="test" options={mockItems} />
    );
    expect(tree).toMatchSnapshot();
  });
});
