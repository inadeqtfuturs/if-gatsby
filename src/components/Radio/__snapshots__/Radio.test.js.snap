// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  className="sc-fznKkj gnrhiW"
>
  <label
    className="sc-fzokOt jOqWMX css-knfhfe"
    required={false}
    size="md"
  >
    test
  </label>
  <div
    className="sc-fznZeY jxyHln"
  >
    <label
      className="css-knfhfe"
      onChange={[Function]}
      style={
        Object {
          "alignItems": "center",
          "fontSize": "0.875rem",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <input
          className="css-p19g2b"
          name="attendance"
          type="radio"
          value="/"
        />
        <svg
          aria-hidden="true"
          className="css-1yqqrb3"
          fill="currentcolor"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="css-1n9usyg"
          fill="currentcolor"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </div>
      home
    </label>
    <label
      className="css-knfhfe"
      onChange={[Function]}
      style={
        Object {
          "alignItems": "center",
          "fontSize": "0.875rem",
        }
      }
    >
      <div
        className="css-vurnku"
      >
        <input
          className="css-p19g2b"
          name="attendance"
          type="radio"
          value="/"
        />
        <svg
          aria-hidden="true"
          className="css-1yqqrb3"
          fill="currentcolor"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 7c-2.76 0-5 2.24-5 5s2.24 5 5 5 5-2.24 5-5-2.24-5-5-5zm0-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="css-1n9usyg"
          fill="currentcolor"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </div>
      nest
    </label>
  </div>
</div>
`;
